src/components/Home/ChangeWallet/styles.module.scss

```diff
@@ -11,11 +11,35 @@
 
   .select{
     width: 100%;
+    height: 50px;
     border-radius: 6px;
     padding: 11px 13px;
+    appearance: none; 
+    border: 1px solid $secondary3;
+    background-color: $bg4;
+    cursor: pointer; 
 
     @media screen and (min-width: map-get($breakpoints, 'tablet')){
       padding: 12px 16px;
     }
+
+    &:focus {
+      outline: none; 
+      border-color: $primary;
+    }
+
+    option {
+      margin-top: 10px;
+      background-color: $bg4; 
+    }
+  }
+
+  &::after {
+    content: '\2039'; 
+    position: absolute;
```

Comment:
why this after ?

Reply:
So there is a problem with styling arrow in select, so we can make custom arrow on wrapper, but i found a new solution to this, and it's more like the default arrow on select.